### PR TITLE
examples(mapbox): use Cube Store for pre-aggregations

### LINKED ISSUE #1334: Epic: Cube Store
This is an Epic issue for initial Cube Store implementation

#### Motivation 
Over the last year, we accumulated feedback among various use cases with pre-aggregations and how to store them.
We figured out that there's a set of problems where relational databases as a storage layer have significant performance and functionality issues.

These problems include high cardinality rollup (1B and more) performance issues, lack of HyperLogLog support, degraded performance for big UNION ALL queries, poor join performance across rolled up tables, and more.

Over time we realized that if we try to fix these issues with existing database engines, we end up modifying these databases' code one way or another.

We decided to go another way and write our own materialized OLAP cache store designed solely to store and serve rollup tables at scale.

To optimize performance as much as possible, we decided to go native and use Rust to develop Cube Store utilizing a set of technologies like RocksDB, Apache Parquet, and Arrow that proved its effectiveness in solving data access problems.

Cube Store will be fully open source and released under the Apache 2.0 license.

The initial release plan is:

- [x] Support most of basic queries available in examples.
- [x] Native builds for popular OS.
- [x] Make Cube Store default external pre-aggregation database.
- [x] Deployment instructions and documentation.

### PR DESCRIPTION
**Pull request by YakovlevCoded**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: examples/mapbox/package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "global": "^4.4.0"
   },
   "devDependencies": {
-    "@cubejs-backend/bigquery-driver": "^0.23.8",
-    "@cubejs-backend/postgres-driver": "^0.23.8",
-    "@cubejs-backend/server": "^0.23.14"
+    "@cubejs-backend/bigquery-driver": "^0.24.5",
+    "@cubejs-backend/postgres-driver": "^0.24.5",
+    "@cubejs-backend/server": "^0.24.5"
```
Comment (igorlukanin) [inline examples/mapbox/package.json:14]: @YakovlevCoded Do we need to install cubestore-driver and remove the unnecessary driver here?

File: examples/external-rollups/package.json [hunk 2]
```diff
@@ -6,8 +6,9 @@
     "dev": "./node_modules/.bin/cubejs-server"
   },
   "devDependencies":  {
-    "@cubejs-backend/bigquery-driver": "^0.23.8",
-    "@cubejs-backend/mysql-driver": "^0.23.11",
-    "@cubejs-backend/server": "^0.23.11"
+    "@cubejs-backend/bigquery-driver": "^0.24.5",
+    "@cubejs-backend/mysql-driver": "^0.24.5",
+    "@cubejs-backend/server": "^0.24.5",
+    "@cubejs-backend/cubestore-driver": "^0.24.5"
```
Comment (igorlukanin) [inline examples/external-rollups/package.json:12]: @YakovlevCoded Do we need to remove the unnecessary driver here?

**Review by igorlukanin — CHANGES_REQUESTED** [review]
@YakovlevCoded Please check that `package.json` files are up to date.

**Comment by hassankhan** [comment]
This PR seems to include changes from https://github.com/cube-js/cube.js/pull/1569, should we close that PR and use this one instead?

**Pull request closed without merging**